From b9fdbeeffc604fdc8260c97e915db2fd1e36708e Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:24:07 -0400
Subject: [cpufreq] intel_pstate: Remove unused member name of cpudata

Message-id: <1409235879-1283-154-git-send-email-prarit@redhat.com>
Patchwork-id: 89492
O-Subject: [RHEL7.1 PATCH BZ 1134639 153/185] cpufreq: intel_pstate: Remove unused member name of cpudata
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 94f89e0760fa7da65c2090c26f0cf59f48221069
Author: Stratos Karafotis <stratosk@semaphore.gr>
Date:   Tue May 20 21:12:27 2014 +0300

    cpufreq: intel_pstate: Remove unused member name of cpudata

    Although, a value is assigned to member name of struct cpudata,
    it is never used.

    We can safely remove it.

    Signed-off-by: Stratos Karafotis <stratosk@semaphore.gr>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org>
    Acked-by: Dirk Brandewie <dirk.j.brandewie@intel.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 24a534a..a6d5afa 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -88,8 +88,6 @@ struct _pid {
 struct cpudata {
  int cpu;
 
- char name[64];
-
  struct timer_list timer;
 
  struct pstate_data pstate;
@@ -544,8 +542,6 @@ static inline void intel_pstate_pstate_decrease(struct cpudata *cpu, int steps)
 
 static void intel_pstate_get_cpu_pstates(struct cpudata *cpu)
 {
- sprintf(cpu->name, "Intel 2nd generation core");
-
  cpu->pstate.min_pstate = pstate_funcs.get_min();
  cpu->pstate.max_pstate = pstate_funcs.get_max();
  cpu->pstate.turbo_pstate = pstate_funcs.get_turbo();
-- 
1.7.1